infra/main.bicep

```diff
@@ -21,6 +21,20 @@ For model limits specific to your region, refer to the documentation at https://
 @maxValue(40)
 param embeddingDeploymentCapacity int = 30
 
+@description('''
+Kernel Memory Service Authorization AccessKey1. It will be passed to the Kernel Memory Service container as an environment variable.
+''')
+@minLength(16)
+@maxLength(42)
+param KernelMemoryServiceAuthorizationAccessKey1 string = 'KernelMemoryServiceAuthorizationAccessKey1'
```

Comment:
I would autogenerate a random value similarly to suffix. The user can still change it, but at least it's safe to begin with

Reply:
in the UI it will look like this  `[substring(newGuid(), 0, 6)]`

Reply:
I'm not sure I understand... 
`suffix` is defined as `substring(newGuid(), 0, 6)` while `KernelMemoryServiceAuthorizationAccessKey1` and `KernelMemoryServiceAuthorizationAccessKey2` are defined as constant values:

```
param suffix string = substring(newGuid(), 0, 6)

param KernelMemoryServiceAuthorizationAccessKey1 string = 'KernelMemoryServiceAuthorizationAccessKey1'

param KernelMemoryServiceAuthorizationAccessKey2 string = 'KernelMemoryServiceAuthorizationAccessKey2'
```

shouldn't these two also use `substring` (and longer than 6 chars) ?

Reply:
I think the best approach is generating a random value, so the deployment is immediately secure - using the same approach used for suffix, just longer than 6 chars